holoviews/plotting/mpl/__init__.py

```diff
@@ -157,7 +158,8 @@ def grid_selector(grid):
 
 
 MPLPlot.sideplots.update({Histogram: SideHistogramPlot,
-                          GridSpace: GridPlot})
+                          GridSpace: GridPlot,
+                          Spikes: MarginalRugPlot})
```

Comment:
Really it's just a subclass with different defaults, so it could be used separately but probably won't be in practice.